kge/config.py

```diff
@@ -235,7 +235,7 @@ def set(
             self.log("Set {}={}".format(key, value))
         return value
 
-    def _import(self, module_name: str):
+    def _import(self, module_name: str, packages: List[str]=["kge"]):
```

Comment:
Can we get rid of the `packages` argument? The config does (or should) have its internal value of the `plugins` option, along the lines of the `import` option.

Reply:
Alright, I had a look at this again.
I think I've come up with a different solution that's nicer, which I will push soon.